pkg/rebuild/maven/infer_test.go

```diff
@@ -17,15 +18,15 @@ import (
 
 type mockMavenRegistry struct {
 	maven.Registry
-	releaseFileContent io.ReadCloser
-	releaseFileError   error
+	artifactCoordinates map[struct{ PackageName, VersionID, FileType string }][]byte
```

Comment:
The reason for `[]byte` here and not `string` (as we store pom XML content) is due to `TestJDKVersionInference` which requires jar bytes as release content.